src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalVariableNameCheck.java

```diff
@@ -61,6 +72,17 @@
  *   &lt;property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   void MyMethod() {
+ *     for (int var = 1; var &lt; 10; var++) {} // OK
+ *     for (int VAR = 1; VAR &lt; 10; VAR++) {} // violation, name 'VAR' must match
+ *                                              // pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
```

Comment:
This 2nd example is the exact same as the first example. Even though the config changed nothing changed between the 2 except the violation message.
Please add another example where with 1 config is a violation and with the other config there is no violation.

Reply:
@jack870131 , please finish this PR.
We understand that might be busy in next few month, so issue completion could wait, but it will be better to finish this PR, as a so little is left to be done.

Reply:
@romani @rnveach Not sure how to deal with this kind of error. It seems like I have something wrong in my indent. 

```
ERROR] testAllCheckSectionJavaDocs(com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest)  Time elapsed: 0.797 s  <<< ERROR!
java.lang.Error: Error was thrown while processing src\main\java\com\puppycrawl\tools\checkstyle\checks\naming\LocalVariableNameCheck.java
	at com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest.examineCheckSection(XdocsJavaDocsTest.java:171)
	at com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest.testAllCheckSectionJavaDocs(XdocsJavaDocsTest.java:144)
Caused by: org.junit.ComparisonFailure: 
LocalVariableName's class-level JavaDoc expected:<...>
class MyClass {
  [ void MyMethod() {
   for (int var = 1; var < 10; var++) {} // OK
   for (int VAR = 1; VAR < 10; VAR++) {} // violation, name 'VAR' must match
                                              // pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
     for (int i = 1; i < 10; i++) {} // OK
   }
 ]}
</pre>
<pre>
class...> but was:<...>
class MyClass {
  [void MyMethod() {
    for (int var = 1; var < 10; var++) {} // OK
    for (int VAR = 1; VAR < 10; VAR++) {} // violation, name 'VAR' must match
                                             // pattern '^[a-z](_?[a-zA-Z0-9]+)*$'
    for (int i = 1; i < 10; i++) {} // OK
  }
]}
</pre>
<pre>
class...>
	at com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest.examineCheckSection(XdocsJavaDocsTest.java:171)
	at com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest.testAllCheckSectionJavaDocs(XdocsJavaDocsTest.java:144)
```

Reply:
@jack870131 It looks like you have an extra space for every line inside the class example.